.docker/docker-qgis-test.sh

```diff
@@ -149,7 +149,7 @@ if [ ${RUN_ORACLE:-"NO"} == "YES" ]; then
 
   echo "Wait a moment while loading Oracle database."
   COUNT=0
-  while ! echo exit | sqlplus -L SYSTEM/adminpass@$QGIS_ORACLETEST_DBNAME &> /dev/null
+  while ! echo exit | sqlplus -L SYSTEM/adminpass@$QGIS_ORACLETEST_DBNAME &> /var/log/oracle.log
```

Comment:
Can you revert these changes?

Reply:
I just thought it might become handy to have this output in case of future failures at hand... but no feelings, I just reverted the file :smile: